svelte-seoul/svelte-admin-ui
# Implement basic installable pwa

## Description

Implement pwa with [vite-plugin-pwa](https://github.com/antfu/vite-plugin-pwa) which automatically generates `service-work` and `manifest` file.


## Test plan
1. install new added packages.  
```
pnpm install
```
2. start `build` and `preview`
```
pnpm build && pnpm preview
```
3. you can check whether service work and manifest files are registered properly or not on the `application` tap.
<img width="1354" alt="Screen Shot 2022-01-10 at 1 15 57 PM" src="https://user-images.githubusercontent.com/58724686/148717915-1ee1318f-1959-496b-85e4-c45e09d46789.png">


4. If there are no errors, you can see an install icon in the address bar depends on your platform.
<img width="833" alt="Screen Shot 2022-01-10 at 1 14 10 PM" src="https://user-images.githubusercontent.com/58724686/148717841-cfbe7841-c7a0-46be-a4a5-319f476fe881.png">
